unit_tests/test_identify_CO_6.py

```diff
@@ -1,4 +1,5 @@
-from scm.plams import Molecule, label
+from scm.plams import Molecule
+from scm.plams.mol.identify import label
```

Comment:
I'm not sure if this import is needed, as `label` is a method of `Molecule`. Same for all other tests in `unit_tests`

Reply:
Same testing showed that, unless explictly imported, the ``label()`` method will not be automatically attached to the ``Molecule`` class. Maybe ebcause it's not specified in ``__all__``?

Reply:
Seems super strange, but ok, if it breaks without it... ;)